knowledge/science/botany/jasminum/qna.yml

```diff
@@ -0,0 +1,26 @@
+created_by: 
+domain: flowers
+seed_examples:
+- answer: | It is  is a species of jasmine native to tropical Asia, from the Indian subcontinent to Southeast Asia.
```

Comment:
Thank you for your submission; taking a look at your `qna.yaml`, we noticed that you have some issues with the multiline `yaml` syntax.

Can you take a look at: https://yaml-multiline.info/ and fix up the file? Thanks!